e2e/nx-firebase-e2e/test-utils/index.ts

```diff
@@ -5,3 +5,391 @@ export * from './test-utils-helpers'
 export * from './test-utils-imports'
 export * from './test-utils-logger'
 export * from './test-utils-project-data'
+
+import { names } from '@nx/devkit'
+import { readJson, runNxCommandAsync } from '@nx/plugin/testing'
+
+const NPM_SCOPE = '@proj'
+
+export interface ProjectData {
+  name: string
```

Comment:
@ciriousjoker the e2e CI isn't passing atm. I don't think your fork is in sync with the latest, since this module is now just a barrel export.

Everything else apart from this looks good, so the simplest fix would be to revert modifications in just this file for this PR?

Reply:
@simondotm Sorry for taking so long, I've moved the changes I made to the old file to the new location. I just added   `--inspect-functions` there